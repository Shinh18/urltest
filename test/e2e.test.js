/* eslint-disable no-undef */
const { run } = require('./run');

describe('end-to-end integration', () => {
  test('when no arguments passed prints error and help message', async () => {
    const { stderr, stdout, exitCode } = await run();
    expect(exitCode).toBe(0);
    expect(stderr).toMatchSnapshot();
    expect(stdout).toEqual('');
  });
  /*
  test('when argument --help is specified should print help message', async () => {
    const { stderr, stdout, exitCode } = await run('--help');
    expect(exitCode).toBe(0);
    expect(stdout).toMatchSnapshot();
    expect(stderr).toEqual('');
  });
  */
});
