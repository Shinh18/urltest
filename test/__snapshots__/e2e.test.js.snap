// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`end-to-end integration when no arguments passed prints error and help message 1`] = ``;
